email_sender/EmailSender.py

```diff
@@ -72,3 +74,14 @@ def get_error_response(ses_response):
         }),
     }
     return response
+
+
+def add_to_suppression_list(emails):
```

Comment:
This function also need not be part of `EmailSender.py`

Reply:
Pushed the cleaned up code